gradoop-examples/gradoop-examples-operators/src/main/java/org/gradoop/examples/common/SocialNetworkGraph.java

```diff
@@ -55,13 +55,13 @@ public static String getGraphGDLString() {
       "(gdbs)-[:hasMember]->(bob)" +
       "(gps)-[:hasModerator {since:2013}]->(dave)" +
       "(gps)-[:hasMember]->(dave)" +
-      "(gps)-[:hasMember]->(carol)-[ckd]->(dave)" +
+      "(gps)-[:hasMember]->(carol)-[ckd0]->(dave)" +
       "(databases)<-[ghtd:hasTag]-(gdbs)-[ghtg1:hasTag]->(graphs)<-[ghtg2:hasTag]-(gps)-[ghth:hasTag]->(hadoop)" +
       "(eve)-[eka:knows {since:2013}]->(alice)-[akb:knows {since:2014}]->(bob)" +
       "(eve)-[ekb:knows {since:2015}]->(bob)-[bka:knows {since:2014}]->(alice)" +
       "(frank)-[fkc:knows {since:2015}]->(carol)-[ckd:knows {since:2014}]->(dave)" +
```

Comment:
In this line, the edge with id ckd is initialized. This declaration has to come before any re-usage of the edge by simply using the id ckd. So instead of renaming both references, the initialization should move to line 58, in 62 just the reference should be used and line 64 should stay unchanged.

Sorry for my first intention to change the identifiers - this seems to be wrong here. Please change it as described and test it with an example that uses this graph. Thx again